token/services/db/sql/postgres/test_utils.go

```diff
@@ -4,7 +4,7 @@ Copyright IBM Corp. All Rights Reserved.
 SPDX-License-Identifier: Apache-2.0
 */
 
-package common
```

Comment:
this test functionality is only relevant to postgres, hence I moved it under the postgres package and removed the postgres prefix.